continuereading.css

```diff
@@ -0,0 +1,617 @@
+body{
+    margin:0;
+    padding:0;
+   
+}
+
+h2{
+    display: flex;
+    font-size:14px;
+    justify-content: center;
+    color: black;
+    margin-top: 20px;
+    margin-left: 20px;
+}
+
+.continuetitle{
+    position: sticky;
+    display: flex;
+    font-size: 80px;
+    font-weight: 700;
+    justify-content: center;
+    font-family: 'Peddana', serif;
+    color:darkorange;
+    margin-top: 50px; 
+    text-decoration: none;
+}
+
+.linebrk{
+    position: sticky;
+    width: 65vw;
+    size: 1px;
+    margin-top: 180px;
+   
+}
+.linebrk1{
+    position: sticky;
+    width: 65vw;
+    size: 1px;
+    margin-top: 1px;
+}
```

Comment:
if you were to use your `border: ...` that I mentioned above, This would be condenced down to about 3 lines of code